### PR TITLE
#25 - Terminate old containers and clean workspace between tests

### PR DESCRIPTION
**Pull request by billvb**
## Pull Request for Issue #25 

**Description of Changes**
 * [x] Stop any lingering docker containers (besides the app itself) between tests.
 * [x] Delete any projects created on disk during the course of the test.

**Exit Criteria**
 - Running tests does not clutter hard drive or CPU usage
 - Each test is cleaned up after it is run. (May need an image prune at the end)

### CODE REVIEW

File: driver.py [hunk 1]
```diff
@@ -94,6 +104,7 @@ def run_playbook(path):
         if any([r[l]['status'].lower() != 'pass' for l in r]):
             failed = True
         full_results[pb] = r
+        stop_project_containers()
```
Comment (mengjiez99) [inline driver.py]: should there be a client variable (docker_client) passed to the stop_project_containers() function?

File: driver.py [hunk 2]
```diff
@@ -8,7 +7,7 @@
 import sys
 import os
 
-# Library imports
+import docker
```
Comment (mgibbs1259) [inline driver.py]: In this PR, should we include docker in requirements.txt?

Comment (billvb) [reply driver.py]: I believe it's alreadyt in requirements.txt

File: driver.py [hunk 3]
```diff
@@ -94,6 +106,10 @@ def run_playbook(path):
         if any([r[l]['status'].lower() != 'pass' for l in r]):
             failed = True
         full_results[pb] = r
+        stop_project_containers(docker.from_env())
```
Comment (mengjiez99) [inline driver.py]: is it better to use docker_client since it was defined earlier

**Pull request merged**